doc/sphinx/source/esmvaldiag/observations.rst

```diff
@@ -0,0 +1,265 @@
+  not be stored in the same file, but in two separate files;
+- NAME: these are the names of the variables you want to extract out of the
+  original data file. The names do not need to be in CMOR standard therefore
+  there is the distinction between ``VAR`` and ``NAME``;
+- MIP: this is the ``mip`` in which the variable is defined (or would be
+  defined if it is not a custom variable or a derived variable) and which
+  describes the realm and the temporal resolution of the dataset; ``Amon`` from
+  the example stands for an atmospheric variable (``A``) in a monthly
+  resolution (``mon``).  **Note:** The description of the MIP is not
+  necessarily structured the same  way as described above. The available
+  choices for MIP are: 3hr, 6hrLev, 6hrPlev, aero, Amon, cf3hr, cfDay, cfMon,
+  cfOff, cfSites, day, fx, grids, LImon, Lmon, Oclim, OImon, Omon, Oyr. See for
+  more details on these different MIPs see the 
+  `CMOR tables <https://github.com/ESMValGroup/ESMValCore/development/esmvalcore/cmor/tables/cmip5/Tables/>`_;
+- FREQ: describes the temporal resolution of the dataset;
+- CMOR_TABLE: provides the link to the CMOR table in which the variable is
+  defined. If the CMOR table is a custom table (like it is here in the example)
+  you need to provide the path and the name of the file in which the definition
+  is stored (here: ``/cmor/tables/custom/CMOR_xch4.dat``). The more basic path
+  information is pulled out of the configuration file (see section 2) that you
+  will have to provide to run the cmorizing script. If your variable is not a
+  custom variable, you would provide here the path to the folder to the table
+  where the variable is available (see for example `cmorize_obs_ERA-Intermim.ncl
+  <https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/utils/cmorizers/obs/cmorize_obs_ERA-Interim.ncl>`_;
+- **Note:** the fields ``VAR``, ``NAME``, ``MIP`` and ``FREQ`` all ask for one
+  or more entries. If more than one entry is provided, make sure that the order
+  of the entries is the same for all four fields! (for example, that the first
+  entry in all four fields describe the variable ``xch4`` that you would like
+  to extract);
+- **Note:** some functions in the script are NCL-specific and are available
+  through the loading of the script ``interface.ncl``. There are similar
+  functions available for python scripts.
+
+In the second part of the script each variable defined in ``VAR`` is separately
+extracted from the original data file and processed. Most parts of the code are
+commented, and therefore it should be easy to follow what is happening. This
+example coded in NCL, since many cmorizing scripts that are available so far
+are written in NCL, and adapting existing code and using existing libraries is
+easier than writing something totally new. However, in theory this script could
+also be written in Python but without the help of the Iris package and rather
```

Comment:
This is probably not correct, many of the current cmorization scripts written in Python use iris. Also, I think it would be better to explain a Python example before the NCL example, because NCL is end of life so new cmorizers should preferably be written in Python.

Reply:
I worked a bit on Section 4 now: I rearranged a few things and I created two subsections, one for an example written in python (which I added), and one for the example in NCL.
@schlunma: could you check if I described the python example correctly, please?